src/simplify_reshapes.cpp

```diff
@@ -183,6 +183,11 @@ struct find_nested_convert
         auto x     = ins->inputs().front();
         auto input = x->inputs().front();
 
+        while(input->name() == "convert")
+        {
+            input = input->inputs().front();
+        }
```

Comment:
Can you add a unit test for this?

Also, if we add a `eliminate_convert` pass, we can probably remove it from here.

Reply:
Can i keep it for some other PR ?

Reply:
opened #2520